### PR TITLE
Logger cleanup

### PR DESCRIPTION
**Pull request by trieloff**
- [x] Remove Loggly
- [x] Add Coralogix
- [x] Add Epsagon

### CODE REVIEW

**Comment by trieloff** [comment]
@stefan-guggisberg I need this to continue with https://github.com/adobe/helix-publish/pull/446

File: src/deploy.cmd.js [hunk 1]
```diff
@@ -421,23 +440,27 @@ Alternatively you can auto-add one using the {grey --add <name>} option.`);
       }
     };
 
-    const params = {
+    const params = Object.entries({
       ...this._default,
-      LOGGLY_HOST: this._loggly_host,
-      LOGGLY_KEY: this._loggly_auth,
-    };
+      EPSAGON_TOKEN: this._epsagonToken,
```
Comment (tripodsan) [inline src/deploy.cmd.js:445]: `HLX_EPSAGON_APP_NAME` is missing

File: test/fixtures/all.env [hunk 2]
```diff
@@ -36,8 +36,6 @@ HLX_FASTLY_AUTH = foobar
 
 # deploy + publish
 HLX_AUTO = true
-HLX_LOGGLY_HOST = my.loggly.com
-HLX_LOGGLY_AUTH = foobar
```
Comment (tripodsan) [inline test/fixtures/all.env:40]: want to add the new ones for testing?

File: test/testDeployCmd.js [hunk 3]
```diff
@@ -438,8 +438,9 @@ describe('hlx deploy (Integration)', () => {
             { key: 'MY_DEFAULT_2', value: 'default-value-2' },
             { key: 'MY_DEFAULT_1', value: 'default-value-1' },
             { key: 'FOO', value: 'bar' },
-            { key: 'LOGGLY_HOST', value: 'loggly-host' },
-            { key: 'LOGGLY_KEY', value: 'loggly-auth' },
+            { key: 'EPSAGON_TOKEN', value: 'fake-token' },
```
Comment (stefan-guggisberg) [inline test/testDeployCmd.js:441]: missing?

```js
{ key: 'EPSAGON_APPLICATION_NAME', value: 'fake-name' },
```

File: src/deploy.cmd.js (hunk 1 above)
Comment (stefan-guggisberg) [inline src/deploy.cmd.js:445]: missing?

```js
EPSAGON_APPLICATION_NAME: this._epsagonAppName,
```

Comment (trieloff) [reply src/deploy.cmd.js:445]: added in c98d03a

File: test/testDeployCmd.js (hunk 3 above)
Comment (trieloff) [reply test/testDeployCmd.js:441]: added in c98d03a

File: src/deploy.cmd.js (hunk 1 above)
Comment (trieloff) [reply src/deploy.cmd.js:445]: added in c98d03a

File: test/fixtures/all.env (hunk 2 above)
Comment (trieloff) [reply test/fixtures/all.env:40]: No.

**Pull request merged**